### PR TITLE
Vungle: Rename from liftoff

### PR DESCRIPTION
**Pull request by Vungle-GordonTian**
https://github.com/prebid/prebid.github.io/pull/5396

### CODE REVIEW

**Comment by SyntaxNode** [comment]
Please also update `adapter_util.go#GetDisabledBidderWarningMessages` to add liftoff as a removed adapter. This allows Prebid Server to understand the adapter used to exist and respond to the publisher with a nice message. You can include the rename to vungle in the message, similar to yssp entry.

**Review by yingchen0706v — APPROVED** [review]

**Comment by Vungle-GordonTian** [comment]
> Please also update `adapter_util.go#GetDisabledBidderWarningMessages` to add liftoff as a removed adapter. This allows Prebid Server to understand the adapter used to exist and respond to the publisher with a nice message. You can include the rename to vungle in the message, similar to yssp entry.

Got it. Thx. BTW, what if there is a new adapter named `liftoff` in the future? Remove this warning?

**Comment by SyntaxNode** [comment]
> Got it. Thx. BTW, what if there is a new adapter named `liftoff` in the future? Remove this warning?

Perhaps. Do you intend to add a liftoff adapter in the future?

**Review by SyntaxNode — DISMISSED** [review]

File: openrtb_ext/imp_liftoff.go
[the remark this answers is not in the document]
Comment (Vungle-GordonTian) [reply openrtb_ext/imp_liftoff.go:1]: Done

[the remark this answers is not in the document]
Comment (SyntaxNode) [reply openrtb_ext/imp_liftoff.go:1]: Good catch. I searched the code for liftoff references, but missed the file names. :)

**Review by SyntaxNode — APPROVED** [review]

**Review by bsardo — APPROVED** [review]

**Comment by bretg** [comment]
I want to remind key folks about the guidelines around renaming.  It's rude to publishers and host companies to pull the rug out from under them be eliminating names suddenly. People might get confused why errors start happening.

Which is why we don't allow abrupt renames like this without a transition period. Liftoff can become an alias of vungle and then at some point in a major release liftoff can be removed

1. please update the code so the vungle adapter lists liftoff as an alias.
2. leave liftoff documentation in place, create vungle docs

**Comment by PBDMSFT** [comment]
Hi @bretg I can't speak for the Liftoff/Vungle team, but from the Microsoft Monetize side, this adapter is not live yet. We've been testing together but, to my knowledge, no one is actively using this adapter. The Liftoff/Vungle team chose to rename their adapter and we have already put in the time to mirror that change on the Monetize side. Unless there are actually any publishers using the adapter through another wrapper, we would strongly prefer to use "Vungle" as that's what we just spent time accommodating on our side.

**Comment by bretg** [comment]
That's good to hear @PBDMSFT , but once something's live in open source, how do we know who's using it?

Unless I get a Vungle/Liftoff person to **guarantee** they have no customers on 'liftoff', Prebid's policy is that the old name has to stay around for a while to give pubs/host companies time to shift.

**Comment by PBDMSFT** [comment]
Understood. Will watch for them to comment. CC @Vungle-GordonTian

**Comment by Vungle-GordonTian** [comment]
> That's good to hear @PBDMSFT , but once something's live in open source, how do we know who's using it?
> 
> Unless I get a Vungle/Liftoff person to **guarantee** they have no customers on 'liftoff', Prebid's policy is that the old name has to stay around for a while to give pubs/host companies time to shift.

I believe `Xandr` is the only one that is testing with us now, and I will let our business PM group to confirm this and reach out to you ASSP. @bretg 
Many thanks for the guidance. @PBDMSFT

**Pull request merged**